README.md

```diff
@@ -8,8 +8,10 @@ Go Lang library to send messages to Slack via Incoming Webhooks.
 ```go
 package main
 
-import "github.com/ashwanthkumar/slack-go-webhook"
-import "fmt"
+import (
+    slack "github.com/ashwanthkumar/slack-go-webhook"
```

Comment:
@JC1738 Is this needed? The package specified in [`main.go`](https://github.com/ashwanthkumar/slack-go-webhook/blob/master/main.go#L1) is slack by default.